zbus/tests/e2e.rs

```diff
@@ -123,6 +123,36 @@ trait MyIface {
 
     #[dbus_proxy(allow_interactive_auth)]
     fn test_interactive_auth(&self) -> zbus::Result<()>;
+
+    #[dbus_proxy(property)]
+    fn modifiable_default(&self) -> zbus::Result<u32>;
+
+    #[dbus_proxy(property)]
+    fn set_modifiable_default(&self, count: u32) -> zbus::Result<()>;
+
+    #[dbus_proxy(property)]
+    fn modifiable_true(&self) -> zbus::Result<u32>;
+
+    #[dbus_proxy(property)]
+    fn set_modifiable_true(&self, count: u32) -> zbus::Result<()>;
+
+    #[dbus_proxy(property)]
```

Comment:
why isn't the proxy side making use of the (existing) `emits_changed_signal` attributes? Same for other methods.